storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -69,6 +69,17 @@ include::modules/lvms-unsupported-devices.adoc[leveloffset=+2]
 
 * xref:../../../storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc#lvms-reference-file_logical-volume-manager-storage[{lvms} reference YAML file]
 
+//Using software RAID arrays
+include::modules/lvms-using-raid-arrays.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../installing/install_config/installing-customizing.adoc#installation-special-config-raid_installing-customizing[Configuring a RAID-enabled data volume]
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_storage_devices/managing-raid_managing-storage-devices#creating-a-software-raid-on-an-installed-system_managing-raid[Creating a software RAID on an installed system]
```

Comment:
RHEL 9 is definetly supported when integrating software RAID arrays with LVM Storage

Reply:
Sorry @sr1kar99 , I meannt this as a question. I did not originally add "?". Would RHEL 8 play any part here? Would inclduing a link to [RHEL 8](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_storage_devices/managing-raid_managing-storage-devices#creating-a-software-raid-on-an-installed-system_managing-raid) also be of value? A lead in sentence such as "For RHEL 8, see []link.

Reply:
In this case, the links serve the purpose of providing more conceptual information about "RAID" arrays. I don't think that a different version link would add much to this purpose. What do you think, @dfitzmau ?

Reply:
Also, we have a proc on "Creating RAID arrays during OCP installation" - which is the first link in the additional resources section. This is the proc that the users must follow to create software RAIDs. The other three links are only for providing additional conceptual info  on RAID arrays.

Reply:
The only issue is if the RHEL 9 guide states something different than the RHEL 8 version. Since the content only targets 4.15 (backporting to earlier versions would be more complex with RHEL compatibility), choosing among RHEL 8 or 9 is easier. If RHEL 9 is correct then no action needed.

Reply:
Maybe move "[Configuring a RAID-enabled data volume](https://70625--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/install_config/installing-customizing#installation-special-config-raid_installing-customizing)" to a Next steps heading?

[Example](https://docs.openshift.com/container-platform/4.14/updating/preparing_for_updates/preparing-manual-creds-update.html#:~:text=or%20service%20accounts.-,Next%20steps,-Update%20the%20upgradeable)

Reply:
Creating software RAID is a prereq in this case. Only if the users have created a RAID array during the OCP installation, they will be able to add it to the LVMCluster. 

I'll loop in the SME to get some clarity on which RHEL version docs would be best here.

@jakobmoellerdev - Could you please confirm which version would be best to add here - RHEL 8 or RHEL 9?

Reply:
Both RHEL versions are supported, since LVMS uses RHEL9 by default starting with 4.14 this should be the goto. Before this we used RHEL8 base images so for older docs this would make sense. 
TLDR: New Stuff should use RHEL9 docs wherever applicable if not otherwise specified

Reply:
Thanks, Jakob. RHEL 9 for 4.15 looks good. @sr1kar99 , LGTM. Feel free to send the PR for merging!